doc/source/examples/incompressible-flow/3d_nitsche_mixer_with_pbt_impeller/nitscher_mixer_with_pbt_impeller.rst

```diff
@@ -0,0 +1,279 @@
+===============================================================================
+3D Mixer with pitched-blade turbine impeller using Nitsche immersed boundary
+===============================================================================
+
+One of the industrial applications which is often simulated with Lethe is mixing in stirred tanks. These agitated vessels are often used in chemical engineering to homogenize products, facilitate heat and mass transfer or to maximize surface area between a catalyst and reactants. They may even be used as a part of a continuous process, in which case we refer to them as CSTR. However, the batch mode of operation is more common in the pharmaceutical and agrofood industries.
+
+This example illustrates how the transient flow in a stirred-tank can be simulated by Lethe using the Nitsche immersed boundary formulation.
+
+Features
+----------------------------------
+- Solvers: ``gls_nitsche_navier_stokes_33``
+- Transient problem
+- Rotating complex solid modeled with Nitsche immersed boundary
+- Calculation of the average velocity
+
+
+Location of the examples
+------------------------
+``/examples/incompressible_flow/3d_nitsche_mixer_with_pbt_impeller/mixer.prm``
+
+
+Description of the case
+-----------------------
+
+We simulate the flow generated by a pitched blade turbine (PBT) in a stirred-tank. The PBT is generally what we refer to as an axial impeller, meaning that it generates a flow pattern that tends to amplify axial circulation within the vessel in opposition to radial impellers (e.g. Rushton turbines) [1].
+
+The setup that we wish to simulate is schematized in the following figure:
+
+  .. image:: images/scheme_tank.png
+   :alt: The geometry and boundary conditions
+   :align: center
+   :name: geometry
+   :height: 12cm
+
+
+
+It consists of a PBT of diameter :math:`D` in a vessel of diameter :math:`T`. The vessel is at a clearance :math:`C` from the bottom. Without loss of generality, we will consider :math:`T=1` and make the problem dimensionless by considering all other dimensions are ratio of :math:`T`. The diameter of the impeller is :math:`D/3` and the clearance is :math:`D/4`. The Reynolds number of the flow will be controlled through the viscosity and the angular velocity of the impeller.
+
+The PBT is rotating at a constant velocity. On the lateral and bottom walls of the vessel, no-slip boundary conditions are applied. On the top surface, a slip boundary condition is applied to mimic the liquid's free surface. This choice of boundary condition is debatable, a do-nothing boundary condition could be used, since free surfaces have a constant pressure, but a slip boundary condition is generally significantly more robust numerically.
```

Comment:
```suggestion
The PBT is rotating at a constant velocity. No-slip boundary conditions are applied On the lateral and bottom walls of the vessel. On the top surface, a slip boundary condition is applied to mimic the liquid's free surface. This boundary condition is debatable. A do-nothing boundary condition could be used since free surfaces have constant pressure, but the slip boundary condition is generally significantly more numerically robust.
```